lib/rules/no-restricted-imports.js

```diff
@@ -83,9 +84,12 @@ module.exports = {
 
         const restrictedPathMessages = restrictedPaths.reduce((memo, importName) => {
```

Comment:
Notice that the restrictedPathMessages will now be an array objects containing a message and an optional objectName.  My new objectName field is used later in ```isRestrictedPath```.